### DISCUSSION TITLE
Introduce 'breakout' alignment for tables

### DISCUSSION THREAD

**Discussion by tosteiner** [knowledgefutures/pubpub#1566] [Ideas]
We're using a growing set of tables to keep track of team members' presentations at conferences, workshops, etc., [see here](https://copim.pubpub.org/pub/copim-outreach-overview/).

As it stands, the table is rather unwieldy, but quite okay to work with 😁 

I noticed that one of PubPub's recent updates introduced the option of setting image alignment to "breakout" so that they break out of the main text body's dimensions, and was wondering if a similar approach could also be implemented for tables, so that tables can also be extended to "escape" the limitations of the main text body?

**Comment by gabestein** [comment]
Thank you once again for the feedback!

We need to spend a sprint or two cleaning up tables in general. A breakout alignment is absolutely something we're thinking about, in addition to better scroll and mobile support.

**Comment by gabestein** [comment]
FYI, should land fairly soon: https://github.com/pubpub/pubpub/pull/2457

**Reply by tosteiner** [reply]
awesome, thanks Gabe!
